Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM public.ecr.aws/docker/library/golang:1.18.4-alpine3.15 as build
+ARG TARGETOS
```

Comment:
These don't seem to be used either via --build-arg or referenced as $TARGETOS - do they have some kind of magic that happens with e go build?

Reply:
yes seems to be crud, i copied this from another guys example. Build a new image without it and confirmed it was still working ok